packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -130,7 +131,10 @@ class WebView extends StatefulWidget {
   ///
   /// A null value is equivalent to an empty set.
   final Set<JavascriptChannel> javascriptChannels;
-
+  
+  /// Enable/Disable zoom.
+  final bool zoom;
```

Comment:
nit: I'd suggest calling this zoomEnabled (I can imagine a parameter named zoom being used to set a zoom level)

Reply:
ok, of course